EagleThreadBot/Program.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System;
 using System.IO;
+using System.Net.Http;
 using System.Threading.Tasks;
+using System.Timers;
 
 using DSharpPlus;
+//using DSharpPlus.Interactivity;
```

Comment:
of course, interactivity cant work if you arent registering the extension.

Reply:
EagleThreadBot/EagleThreadBot.cs